src/Components/Editor/Editor.jsx

```diff
@@ -40,64 +44,42 @@ const Editor = () => {
 		<section className={styles.editorSection}>
 			<h3 className={styles.editorTitle}>New Note Title</h3>
 			<ReactQuill
+				className={`${styles.quill} ${bgColorCheck(bgColor)}`}
 				theme="snow"
-				value={noteTitle}
-				onChange={(e) => setNoteTitle(e)}
+				value={newNote.title}
+				onChange={(e) => setNewNote((prev) => ({ ...prev, title: e }))}
 				modules={titleModules}
 			/>
 			<h3 className={styles.editorTitle}>Content</h3>
 			<ReactQuill
-				style={{ backgroundColor: bgColor }}
+				className={`${styles.quill} ${bgColorCheck(bgColor)}`}
 				theme="snow"
-				value={noteContent}
-				onChange={(e) => setNoteContent(e)}
+				value={newNote.content}
+				onChange={(e) => setNewNote((prev) => ({ ...prev, content: e }))}
```

Comment:
you could make this code more readable by changing the ```e``` to ```value``` as react quill takes ```value``` default argument.

Reply:
since it's the default `event` object provided by `eventlistener` I prefer it to be `e`. Setting it to `value` might confuse later.